### ISSUE TITLE
Visualization Demo

### ISSUE THREAD

**Issue by stan4lv** [gy20073/BDD_Driving_Model#16]
Dear authors, I am a newbie on python. I have a difficulty on visualizing the the result from the single image with 'generate_visualization(img, logits, method="vis_continuous")'. What's is logit parameter? Also, how we called this parameter?

THanks
